src/simulation/dynamics/prescribedMotion/prescribedMotion.rst

```diff
@@ -0,0 +1,96 @@
+
+Executive Summary
+-----------------
+
+The prescribed motion class is an instantiation of the state effector abstract class. Two integrated tests are performed

```

Comment:
The summary should discuss the motivation for the module and what it down. Don't discuss the unit tests here.  Please scratch this paragraph and rewrite.

Reply:
Please let me know if the updated summary is satisfactory @schaubh